de.haeckerfelix.Shortwave.json

```diff
@@ -1,50 +1,79 @@
 {
-    "app-id" : "de.haeckerfelix.Shortwave",
-    "runtime" : "org.gnome.Platform",
-    "runtime-version" : "45",
-    "sdk" : "org.gnome.Sdk",
-    "sdk-extensions" : ["org.freedesktop.Sdk.Extension.rust-stable"],
-    "command" : "shortwave",
-    "finish-args" : [
+    "id": "de.haeckerfelix.Shortwave",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "shortwave",
+    "finish-args": [
         "--share=network",
         "--share=ipc",
         "--socket=fallback-x11",
         "--socket=wayland",
         "--device=dri",
         "--socket=pulseaudio",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
         "--filesystem=xdg-music"
     ],
-    "build-options" : {
-        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
     },
-    "modules" : [
+    "modules": [
+        {
+            "name": "glycin-loaders",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dglycin-loaders=true",
+                "-Dlibglycin=false"
+            ],
+            "sources": [
+                {
+                    "type": "git",
```

Comment:
If you are using the git version here, you also have to use flatpak-builder-tools for this project.

Reply:
Makes sense, thanks!